engine/installation/linux/debian.md

```diff
@@ -105,20 +99,32 @@ Docker from the repository.
       uid                  Docker Release Tool (releasedocker) <docker@docker.com>
     ```
 
-3.  Use the following command to set up the **stable** repository. To also
-    enable the **testing** repository, add the words `testing` after `main` on
-    the last line.
-    **Do not use these unstable repositories on production systems or for non-testing workloads.**
+3.  Use the following command to set up the **stable** repository.
 
-    ```bash
-    $ sudo add-apt-repository \
-           "deb https://apt.dockerproject.org/repo/ \
-           debian-$(lsb_release -cs) \
-           main"
-    ```
+    > **Note**: The `lsb_release -cs` sub-command below returns the name of your
+    > Debian distribution, such as `jessie`.
+
+    - **Debian**:
+
+      ```bash
+      $ sudo add-apt-repository \
+             "deb https://apt.dockerproject.org/repo/ \
+             debian-$(lsb_release -cs) \
+             main"
+      ```
 
-    To disable the `testing` repository, you can edit `/etc/apt/sources.list`
-    and remove the word `testing` from the appropriate line in the file.
+    - **Raspbian**:
+
+      ```bash
+      $ sudo add-apt-repository \
+             "deb https://apt.dockerproject.org/repo/ \
+             raspbian-$(lsb_release -cs) \
+             main"
+      ```
+
+    To enable the `testing` repository, you can edit `/etc/apt/sources.list`
+    and add the word `testing` after `main` on the appropriate line of the file.
+    **Do not use these unstable repositories on production systems or for non-testing workloads.**
```

Comment:
Non-blocking:  to match the cent OS version of this warning, maybe we can change "these unstable repositories" to just "unstable repositories"?  When I read this the first time I thought "main" was also being referred to as an unstable repository because of the word "these".